docs/en/stack/ml/df-analytics/ml-dfa-phases.asciidoc

```diff
@@ -0,0 +1,76 @@
+[role="xpack"]
+[[ml-dfa-phases]]
+=== How a {dfanalytics-job} works
+
+This page provides high-level background information on how {dfanalytics-jobs} 
+work, what kind of phases the jobs have, and what happens during the individual 
+phases.
+
+
+[discrete]
+==== The phases of a {dfanalytics-job}
+
+A {dfanalytics-job} has two states: started and stopped. Until its state 
+switches from started to stopped, the job goes through four phases regardless of 
+the {dfanalytics} type it performs. The phases are discrete steps of the 
+{dfanalytics-job} that is essentially a persistent {es} task. The four phases 
+are the following:
+
+* reindexing,
+* loading data,
+* analyzing,
+* writing results.
+
+Let's take a look at the phases one-by-one.
+
+
+[discrete]
+===== Reindexing
+
+During the reindexing phase the documents from the source index or indices are 
+copied to the destination index. If the destination index does not exist, then 
+the job creates it before reindexing by using default settings. If it is already 
+created, then the job will use the existing index. This provides flexibility for 
+you to create the destination index and define any settings or mappings that you 
+want to have in it. 
+
+If the destination index is built, the {dfanalytics-job} task calls the {es} 
```

Comment:
Perhaps replace `If` with `Once`? The job will call the reindex API regardless of whether it had to create the destination index or not.